src/index.html

```diff
@@ -8,10 +9,52 @@
 
   <link rel="stylesheet" href="css/main.min.css" />
 </head>
+
 <body>
-  <h1>Hello, World!</h1>
-  <h2>Yarik lox</h2>
+  <header>
+
+    <div class="container">
+      <div class="intro">
+        <div class="circle"></div>
+        <div class="intro_name">
+          <span>Themebau.</span>
+        </div>
+
+        <div class="navigation">
+          <nav class="nav">
+            <ul>
+              <li><a href="#">home</a></li>
+              <li><a href="#">projects</a></li>
+              <li><a href="#">pages</a></li>
+              <li><a href="#">contact</a></li>
+            </ul>
+          </nav>
+        </div>
+
+
+        <div class="social_links">
+          <a href="#"><img src="img/facebook.svg" alt="Facebook logo"></a>
+          <a href="#"><img src="img/twitter.svg" alt="Twitter logo"></a>
+          <a href="#"><img src="img/inst.svg" alt="Instagram logo"></a>
+        </div>
+
+        <div class="intro_menu">
+          <span></span>
+          <span></span>
+          <span></span>
+        </div>
+
+      </div>
+
+      <div class="intro_info">
+        <h2 class="intro_info_title">An award-winning digital studio ✌️</h2>
+        <a href="#" class="btn">let`s work<span></span></a>
```

Comment:
Replace `span` based circle to after element based

Reply:
Done